### PR TITLE
Update forms library (part 2)

### PR DESCRIPTION
**Pull request by pinussilvestrus**
## Description

This is a follow up of #2169. This includes the features that won't be part of the next alphas, so we can wait with reviewing it.

## When should this change go live?

- [ ] This change is not yet live and should not be merged until {ADD_DATE} (apply `hold` label or convert to draft PR)?
- [ ] There is no urgency with this change.
- [ ] This change or page is part of a marketing blog, conference talk, or something else on a schedule.
- [ ] This functionality is already available but undocumented.
- [ ] This is a bug fix or security concern.

## PR Checklist

- [ ] I have added changes to the relevant `/versioned_docs` directory, or they are not for an **already released version**.
- [x] I have added changes to the main `/docs` directory (aka `/next/`), or they are not for **future versions**.
- [x] My changes require an [Engineering review](https://github.com/camunda/camunda-platform-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned an engineering manager or tech lead as a reviewer, or my changes do not require an Engineering review.
- [x] My changes require a [technical writer review](https://github.com/camunda/camunda-platform-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @christinaausley as a reviewer, or my changes do not require a technical writer review.

### CODE REVIEW

**Comment by pinussilvestrus** [comment]
Update: as this didn't make it into the alpha3 release, I will open this up for review after the release.

**Comment by pinussilvestrus** [comment]
This is ready for review now 👍

**Review by vsgoulart — APPROVED** [review]

File: docs/components/modeler/forms/configuration/forms-config-options.md [hunk 1]
```diff
@@ -43,6 +43,10 @@ The expected format for the data is an array of options, each defining its label
 }
 ```
 
+### FEEL expressions
+
+The options are populated from a [FEEL expression](../../feel/language-guide/feel-expressions-introduction.md) that returns an array of options. The resulting array must be formatted as described in the [as above](#input-data-driven-by-process-data).
```
Comment (christinaausley) [inline docs/components/modeler/forms/configuration/forms-config-options.md:48]: "as described in the [as above]" -- what exactly are you referring to here?

**Review by christinaausley — CHANGES_REQUESTED** [review]
Just one comment for clarification here.

File: docs/components/modeler/forms/configuration/forms-config-options.md (hunk 1 above)
Comment (pinussilvestrus) [reply docs/components/modeler/forms/configuration/forms-config-options.md:48]: The "as above" is referring to the example JSON for the input data.

Comment (christinaausley) [reply docs/components/modeler/forms/configuration/forms-config-options.md:48]: Okay -- I would likely reword as "The resulting array must be formatted as described in the [example JSON above]." Does this make more sense?

Comment (pinussilvestrus) [reply docs/components/modeler/forms/configuration/forms-config-options.md]: Agreed, I pushed the change 👍

**Review by christinaausley — APPROVED** [review]

**Comment by christinaausley** [comment]
@pinussilvestrus Feel free to squash and merge when you are ready 👍

**Pull request merged**